promise.js

```diff
@@ -0,0 +1,63 @@
+// Setteled Promise :- A promise is said to be settled if it is either fulfilled or rejected, but not pending.
+
+let promise = new Promise(function (resolve, reject) {
+    const x = 1;
+    const y = 1;
+    if (x === y) {
+        resolve(4);
+    } else {
+        reject();
+    }
+});
+
+let promise1 =  Promise.resolve(6)
+let promise2 = new Promise(function(resolve,){
```

Comment:
variable names should not contain numerics.

Reply:
change the variable name